openedx/features/edly/api/v1/views/user_sites.py

```diff
@@ -0,0 +1,60 @@
+"""
+Views for user sites API
+"""
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.mobile_api.decorators import mobile_view
+from openedx.features.edly.api.serializers import UserSiteSerializer
+
+
+@mobile_view()
+class UserSitesViewSet(viewsets.ViewSet):
+    """
+    **Use Case**
+
+        Get information about the current user's sites and mobile configurations
+        of the sites with which user is linked.
+
+        Apps hit this endpoint after obtaining access token for a user.
+
+        You can use the **app_configs** value in the response to get a
+        mobile configurations for the sites with user is linked.
+
+        You can use the **site_data** value in the response to get a
+        branding configurations for the sites with user is linked.
+
+    **Example Request**
+
+        GET /api/mobile/v1/user-sites/
+
+    **Response Values**
+
+        If the request is successful, the request returns an HTTP 200 "OK" response.
+
+        The HTTP 200 response has the following values.
+
+        * app_config: Mobile configurations for the site with user is linked.
+        * site_data: Branding configurations for the site with user is linked.
```

Comment:
```python
* app_config: Mobile configurations for the site which user is linked with.
* site_data: Branding configurations for the site which user is linked with.
```

Reply:
It will return the array of sites data having these two fields. Doc should be updated accordingly.

Reply:
@farhan edX user details API also used the same format.